lib/api/graphql/query/surveys_query.dart

```diff
@@ -0,0 +1,18 @@
+const String GET_SURVEYS_QUERY = r"""
+  query($endCursor: String){
```

Comment:
@sleepylee just curious do we have any documentation for graphql? 🤤

Reply:
do you mean graphql_flutter or our graphql api? 🤔

Reply:
ah I mean our graphql api 😃

Reply:
@manh-t you can use `Altair` app as suggested. Our GraphQL API will provide the specs (like postman) once you connect to it
![image](https://user-images.githubusercontent.com/13435717/123577348-dda1d380-d7fd-11eb-8d12-c1eea338fc62.png)

![image](https://user-images.githubusercontent.com/13435717/123577395-ef837680-d7fd-11eb-87af-ea72d8dc63a2.png)

Reply:
I found a bug 😆 after setting the variable to `endCursor` in the query string, I have to pass it here instead of the actual variable `after:`

Reply:
Thanks for sharing 💴